### PR TITLE
DOC-1572: Updated doc for various options

### PR DESCRIPTION
**Pull request by tiny-ben-tran**
Related Ticket: DOC-1572

Description of Changes:
* Removed `media_scripts` doc
* Removed the version based section in the `forced_root_block` doc
* Updated `image_upload_handler()` in the local upload live demo

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] ~`_data/nav.yml` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by lnewson — CHANGES_REQUESTED** [review]
Looks like https://github.com/tinymce/tinymce-docs/blob/staging/docs-6/modules/ROOT/examples/live-demos/local-upload/index.js#L10 was missed and this needs updating. The rest looks good though 👍

**Review by EkimChau — APPROVED** [review]

File: modules/ROOT/examples/live-demos/local-upload/index.js [hunk 1]
```diff
@@ -7,11 +7,11 @@ tinymce.init({
...
-  images_upload_handler: (blobInfo, success, failure) => {
-    setTimeout(() => {
+  images_upload_handler: (blobInfo) => {
+    return new Promise((resolve) => {
       /* no matter what you upload, we will turn it into TinyMCE logo :)*/
-      success('http://moxiecode.cachefly.net/tinymce/v9/images/logo.png');
-    }, 2000);
+      resolve('http://moxiecode.cachefly.net/tinymce/v9/images/logo.png');
+    });
   },
```
Comment (lnewson) [inline modules/ROOT/examples/live-demos/local-upload/index.js:15]: This conversion is not correct... you've dropped the `setTimeout`. Also the `new Promise` part should be collapsed so it's on the same line, e.g. `(blobInfo) => new Promise(resolve) => {`

**Review by lnewson — CHANGES_REQUESTED** [review]

File: modules/ROOT/examples/live-demos/local-upload/index.js [hunk 2]
```diff
@@ -7,11 +7,9 @@ tinymce.init({
...
 
   /* we override default upload handler to simulate successful upload*/
-  images_upload_handler: (blobInfo, success, failure) => {
-    setTimeout(() => {
+  images_upload_handler: (blobInfo) => new Promise((resolve) => {
       /* no matter what you upload, we will turn it into TinyMCE logo :)*/
-      success('http://moxiecode.cachefly.net/tinymce/v9/images/logo.png');
-    }, 2000);
-  },
+      setTimeout(() => resolve('http://moxiecode.cachefly.net/tinymce/v9/images/logo.png'), 2000);
```
Comment (lnewson) [inline modules/ROOT/examples/live-demos/local-upload/index.js:12]: This still isn't a correct conversion yet sorry Ben... it should be this:
```
  images_upload_handler: (blobInfo) => new Promise((resolve) => {
      setTimeout(() => {
        /* no matter what you upload, we will turn it into TinyMCE logo :)*/
        resolve('http://moxiecode.cachefly.net/tinymce/v9/images/logo.png')
      }, 2000);
  }),
```

Comment (tiny-ben-tran) [reply modules/ROOT/examples/live-demos/local-upload/index.js:12]: No problem. It seems more readable

**Review by lnewson — APPROVED** [review]

File: modules/ROOT/examples/live-demos/local-upload/index.js (hunk 2 above)
Comment (lnewson) [reply modules/ROOT/examples/live-demos/local-upload/index.js]: Changes look god now thanks!

**Pull request merged**